addons/point_of_sale/views/pos_config_view.xml

```diff
@@ -119,6 +119,7 @@
                                 <label for="is_posbox"/>
                                 <div class="text-muted">
                                     Use an integrated hardware setup like <a href="https://www.odoo.com/documentation/user/online/point_of_sale/overview/setup.html" target="_blank">IoT Box</a>
+                                    <br/><br/>If using HTTPS, make sure to first validate the certificate of the IoTBox by navigating to https://proxy-ip
                                 </div>
```

Comment:
@twa-odoo Could we also test in the PoS somehow if there is a problem with the certificate and return the error there?